programs/openbook_v2/tests/integration.rs

```diff
@@ -0,0 +1,319 @@
+use std::str::FromStr;
+use solana_program::instruction::{AccountMeta, Instruction};
+use solana_program_test::{ tokio, ProgramTest, ProgramTestBanksClientExt };
+use solana_sdk::{
+    signer::Signer,
+    transaction::Transaction,
+};
+use anchor_lang::{AccountDeserialize, Accounts, Key};
+use anchor_lang::InstructionData;
+use bytemuck::Zeroable;
+use solana_program::program_pack::Pack;
+use solana_program::system_program;
+use openbook_v2_light::{ PlaceOrderType, SelfTradeBehavior, Side };
+use openbook_v2_light::instruction::PlaceOrder;
+use drift::controller::position::PositionDirection;
+use drift::instruction::{Deposit};
+use drift::instructions::SpotFulfillmentType;
+use drift::state::order_params::OrderParams;
+use drift::state::user::{MarketType, OrderType, User};
+use drift::state::user::OrderTriggerCondition::{TriggeredAbove, TriggeredBelow};
+use pyth::instruction::Initialize as PythInitialize;
+
+use crate::drift_utils::{create_user, deposit_and_execute, init_quote_market, init_spot_market, initialize_drift, initialize_openbook_v2_config, place_spot_order_and_execute};
+use crate::program::{setup_programs};
+use crate::market::{create_bids_asks_event_heap, create_default_market, MarketKeys, place_order_and_execute};
+use crate::ooa::setup_open_orders_account;
+use crate::pyth_utils::initialize_pyth_oracle;
+use crate::token::init_mint;
+use crate::utils::get_paths;
+
+mod program;
+mod utils;
+mod market;
+mod ooa;
+mod drift_utils;
+mod token;
+mod pyth_utils;
+
+#[tokio::test]
+async fn test_program() -> anyhow::Result<()>{
+    let mut validator = ProgramTest::default();
```

Comment:
would probably just load drift program in here on init with `ProgramTest::new()` instead of `add_programs`

Reply:
done